index.html

```diff
@@ -3447,6 +3447,19 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+
+      <section>
+        <h3>Mappings for Verifiable Credentials</h3>
+
+        <p>
+The standard representation of a Verifiable Credential is defined in this specification (which uses the media type <code>application/vc+ld+json</code>).
+Other representations can be used, provided that a mapping is defined between that representation and the standard one.
```

Comment:
In order to avoid confusion, we should say that the required mapping must be for all instances in a given representation to the standard one (i.e., at least one "general" mapping exists). A bespoke, one-off mapping for some particular instance in that representation is insufficient.

Of course, anyone can use whatever mappings they want to (including doing bespoke mappings) for application-specific purposes, but that is orthogonal to the need for a general purpose mapping.

```suggestion
Other representations can be used, provided that a mapping is defined for all
instances in that representation to the standard one.
```

Reply:
I can live this this, although I don't know that everyone understands existential qualifiers.

Reply:
there could be more than one mapping per representation, as long as it maps back to the same base data model, so I do not understand `for all` part.

Reply:
```suggestion
Other representations can be used, provided that a mapping is defined for any
instances in that representation to the standard one.
```

i can live with `for any` I think.

Reply:
What I want us to say is: for all `x` in some other representation, there must be an `f` such that `f(x)` is the standard representation. This *might* be the same as saying "for any `x`" and we're just stuck on a language issue but really mean the same thing. What is important is that there must not be any `x` such that `f(x)` is undefined; `f` must be a complete mapping ... which rules out handcrafted mappings as being sufficient here.

Again, this doesn't mean someone can make a handcrafted mapping for some particular use case (not that we could ever stop that with spec text anyway), nor does it mean that there can't be some other function `g` with the same properties as `f`. It just means that at least some `f` must be defined.